addons/point_of_sale/static/tests/tours/pos_cash_rounding_tour.js

```diff
@@ -0,0 +1,307 @@
+import * as Chrome from "@point_of_sale/../tests/tours/utils/chrome_util";
+import * as Dialog from "@point_of_sale/../tests/tours/utils/dialog_util";
+import * as ProductScreen from "@point_of_sale/../tests/tours/utils/product_screen_util";
+import * as PaymentScreen from "@point_of_sale/../tests/tours/utils/payment_screen_util";
+import * as ReceiptScreen from "@point_of_sale/../tests/tours/utils/receipt_screen_util";
+import { registry } from "@web/core/registry";
+
+registry
+    .category("web_tour.tours")
+    .add("test_cash_rounding_halfup_add_invoice_line_not_only_round_cash_method", {
+        steps: () =>
+            [
+                Chrome.startPoS(),
+                Dialog.confirm("Open Register"),
+
+                // First order.
+                ProductScreen.addOrderline("random_product", "1"),
+                ProductScreen.clickPartnerButton(),
+                ProductScreen.clickCustomer("AAAAAA"),
+                ProductScreen.clickPayButton(),
+
+                PaymentScreen.totalIs("15.70"),
+                PaymentScreen.clickPaymentMethod("Cash", true, { remaining: "0.0" }),
+
+                PaymentScreen.clickInvoiceButton(),
+                PaymentScreen.clickValidate(),
+
+                ReceiptScreen.receiptAmountTotalIs("15.72"),
+                ReceiptScreen.receiptRoundingAmountIs("-0.02"),
+                ReceiptScreen.receiptToPayAmountIs("15.70"),
+                ReceiptScreen.receiptChangeAmountIsNotThere(),
+                ReceiptScreen.clickNextOrder(),
+
+                // Second order.
+                ProductScreen.addOrderline("random_product", "1"),
+                ProductScreen.clickPartnerButton(),
+                ProductScreen.clickCustomer("AAAAAA"),
+                ProductScreen.clickPayButton(),
+
+                PaymentScreen.totalIs("15.70"),
+                PaymentScreen.clickPaymentMethod("Bank"),
+                PaymentScreen.clickNumpad("."),
+                PaymentScreen.remainingIs("15.70"),
+                PaymentScreen.clickNumpad("6 7"),
```

Comment:
For some reason, `PaymentScreen.clickNumpad("0 . 6 7")` doesn't work. The docstring says:
```
/**
 * Press the numpad in sequence based on the given space-separated keys.
 * NOTE: Maximum of 2 characters because NumberBuffer only allows 2 consecutive
 * fast inputs. Fast inputs is the case in tours.
 *
 * @param {...String} keys space-separated numpad keys
 */
```
I haven't find any better way to do that. If you have a better suggestion, let me know

Reply:
```suggestion
                PaymentScreen.clickNumpad(". 6"),
                PaymentScreen.remainingIs("15.10"),
                PaymentScreen.clickNumpad("7"),
```

can you try this?

Reply:
It's better to change it to please the tour runner.